plaidml/op/lib/ops.cc

```diff
@@ -2163,8 +2191,12 @@ Value nms(const Value& value) {
     New_scores = edsl::reshape(New_scores_update, {num_batches, num_classes, num_boxes});
 
     Tensor IOU_candidate = edsl::gather(IOU, Candidate_index).mode(edsl::GatherMode::ND).batchDims(1);
-    // use >= to include suppose_hard_suppresion case
-    New_scores = edsl::select(IOU_candidate >= IOU_threshold_i, Zero, New_scores);
+    // use >= to include aaasuppose_hard_suppresion case
+    if (nms_style == NmsStyle::MXNET) {
+      New_scores = edsl::select(op::abs(IOU_candidate - IOU_threshold_i) < 1e-7, Zero, New_scores);
```

Comment:
epsilon as 1e-7 is fine for float32, but the doc says the type can be any type of float, may change or add comments here

Reply:
Yes this is a workaround to test the feasibility of doing so. I'm not sure what precision we should use here exactly though.